I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 285, in script
    extend " I feel a pinch and a strange pressure in my chest."
  File "renpy/common/00library.rpy", line 132, in extend
    renpy.exports.say(who, what, interact=interact)
IOError: Couldn't find file 'music/Infocalyse_-_Harvest.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 285, in script
    extend " I feel a pinch and a strange pressure in my chest."
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\ast.py", line 613, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\exports.py", line 1147, in say
    who(what, interact=interact)
  File "renpy/common/00library.rpy", line 132, in extend
    renpy.exports.say(who, what, interact=interact)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\exports.py", line 1147, in say
    who(what, interact=interact)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\character.py", line 877, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\character.py", line 716, in do_display
    **display_args)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\character.py", line 508, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\display\core.py", line 2526, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\display\core.py", line 2825, in interact_core
    renpy.audio.audio.interact()
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 930, in interact
    periodic()
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 864, in periodic
    c.periodic()
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 403, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\audio\audio.py", line 67, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\john\Downloads\renpy-6.99.12.4-sdk\renpy\loader.py", line 543, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'music/Infocalyse_-_Harvest.mp3'.

Windows-7-6.1.7601-SP1
Ren'Py 6.99.12.4.2187
Manna for our Malices 1.0
